GOVERNANCE.md

```diff
@@ -0,0 +1,36 @@
+# Governance
+
+`haskell-cryptography` (herein referred to as the "Organisation") is a GitHub organisation that aims to provide a support structure to open-source maintainers producing cryptography libraries.
+
+This document elaborates on administrative and operational aspects of the Organisation.
+
+## Administrative aspect
+
+The Organisation's administrative aspects are handled by the Core Team. While a healthy overlap is appreciated between the Core Team and the contributors,
+An employee of the Haskell Foundation has a position in the Core Team as to facilitate ecosystem-related matters and discussion with other institutional parties.
+
+## Operational aspect
+
+As an Organisation, `haskell-cryptography` concerns itself with two things:
+
+1. Code production
+2. Services related to the code and users
+
+### Code production
+
+`haskell-cryptography` produces a series of projects related to cryptography.
+These projects have a referent person that is in charge of being the main point of contact, handle inquiries from the core team and act as an editor for the project.
+
+This role involves making concrete the strategy of the core team, and being in charge of the management of the project,
+whithin the framework established by the guidelines published in the `haskell-cryptography/governance` repository.
+The referent person is  autonomous but held accountable. This doesn't prevent a decision proces that is involving multiple parties in a "flat"
+governance structure for a project, but the referent is the one person that is in charge of making these decisions official, answering for them and enforcing them.
+
```

Comment:
So, I wrote 

> This role involves making concrete the strategy of the core team, and being in charge of the management of the project, within the framework established by the guidelines published in the haskell-cryptography/governance repository.

In simplistic terms, this would be a subjugation relationship? But I think this implies no agency whatsoever, so it's the wrong term.

What I'm trying to say is that the Core Team establishes a strategy (improve the usability and quality of cryptography toolkits in the ecosystem, collaborate with industrial users to better answer their needs, something like that), and the persons that are referent for projects should aim to enact this strategy at their level, and they can ask for resources from the Core Team (for example, provide with a list of industrial users' needs, get in-kind donations from companies, etc).

Reply:
@axman6 Could you take a look again? I hope I've clarified the situation?

Reply:
This looks good, nice change.

Reply:
I've taken your feedback into account. I'll let you formally approve or request changes from the GH interface when you have time. :)